test/functional/editor_controller_test.rb

```diff
@@ -45,8 +45,10 @@ def setup
         tags: 'question:question,one',
         template: 'question',
         redirect: 'question'
-    assert_select "h3", "Ask a question of the community"
-    assert_select "input#taginput[value=?]", "question:question,one"
+        assert_response :redirect
```

Comment:
I worry about details too :-) So all the assert lines should line up? And do you want me to actually delete the 2 lines I commented out:
```
assert_select "h3", "Ask a question of the community"
assert_select "input#taginput[value=?]", "question:question,one"
```
I commented them out just cause the original issue didn't seem clear about whether or not they should be out or not.